NISTIR_Metadata.md

```diff
@@ -10,7 +10,7 @@ Oftentimes, a set of asserted attributes and their values is enough on its own t
 
 ![](media/Canvas 2.png)
 
-The attribute schema metadata and attribute value metadata listed in each section of this schema are not mandatory. Collectively, these elements aim to support transactional needs in federations and trust frameworks across the private and public sectors. NIST envisions that communities and federations will leverage the included metadata elements to develop their own profiles.
+The attribute schema metadata and attribute value metadata listed in each section of this schema are not mandatory. Collectively, these elements aim to support transactional needs in federations and trust frameworks across the private and public sectors. NIST envisions that communities and federations will leverage the included metadata elements to develop their own profiles. In addition, this schema supports extensibility for instances when the information contained herein is not sufficient for federated partners.

```

Comment:
does it support extensibility, or is it expected that it would be expanded to support the needs of federated partners?